src/core/IconButton/index.stories.tsx

```diff
@@ -28,72 +29,158 @@ export default {
 
 const Template: Story = (args) => <Demo {...args} />;
 
-export const LargePrimaryIconButton = Template.bind({});
+export const Default = Template.bind({});
 
-LargePrimaryIconButton.args = {
+Default.args = {
   "aria-label": "info",
   disabled: false,
   icon: <Icon sdsIcon="infoCircle" sdsSize="xl" sdsType="iconButton" />,
   sdsSize: "large",
   sdsType: "primary",
 };
 
-export const LargeSecondaryIconButton = Template.bind({});
-
-LargeSecondaryIconButton.args = {
-  "aria-label": "checked",
-  disabled: false,
-  icon: <Icon sdsIcon="checkCircle" sdsSize="xl" sdsType="iconButton" />,
-  sdsSize: "large",
-  sdsType: "secondary",
+Default.parameters = {
+  snapshot: {
+    skip: true,
+  },
 };
 
-export const LargeTertiaryIconButton = Template.bind({});
+export const Test = Template.bind({});
 
-LargeTertiaryIconButton.args = {
-  "aria-label": "x mark",
+Test.args = {
+  "aria-label": "info",
   disabled: false,
-  icon: <Icon sdsIcon="xMark" sdsSize="xl" sdsType="iconButton" />,
+  icon: <Icon sdsIcon="search" sdsSize="xl" sdsType="iconButton" />,
   sdsSize: "large",
-  sdsType: "tertiary",
-};
-
-export const MediumTertiaryIconButton = Template.bind({});
-
-MediumTertiaryIconButton.args = {
-  "aria-label": "x mark",
-  disabled: false,
-  icon: <Icon sdsIcon="xMark" sdsSize="l" sdsType="iconButton" />,
-  sdsSize: "medium",
-  sdsType: "tertiary",
+  sdsType: "secondary",
 };
 
-export const SmallPrimaryIconButton = Template.bind({});
+const LivePreviewDemo = (props: Args): JSX.Element => {
+  const spacings = getSpaces(props);
```

Comment:
Since we have the `defaultTheme` object, we can retrieve `spaces` from it directly.

```
const spacings = defaultAppTheme?.spacing;
```